File: benchmark/fib.js
function fib(n) {
  if (n < 2)
    return n;
  return fib(n - 1) + fib(n - 2);
} 

const n = process.argv[2];
const m = process.argv[3];
for (let i = 0; i < n; i++)
  console.log(fib(m));

File: benchmark/fib.rb
def fib(n)
  if n < 2 then
    return n
  end
  fib(n - 1) + fib(n - 2)
end

n = ARGV[0].to_i
m = ARGV[1].to_i
for i in 0...n
  puts fib(m)
end
